src/data-access/repositories/mongo/mongo-core-device-repository.ts

```diff
@@ -12,6 +12,11 @@ export class MongoCoreDeviceRepository extends BaseMongoRepository implements De
   constructor(mongoDatabase: MongoDatabase, private readonly mongoEntityConverter: MongoEntityConverter) {
     super(mongoDatabase)
   }
+  
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Since this is targeting develop, then this shouldn't be necessary, since `_` prefixed parameters are now allowed there :)